sklearn/discriminant_analysis.py

```diff
@@ -554,6 +554,13 @@ class QuadraticDiscriminantAnalysis(BaseEstimator, ClassifierMixin):
         Regularizes the covariance estimate as
         ``(1-reg_param)*Sigma + reg_param*np.eye(n_features)``
 
+    store_covariances : boolean, (default False)
```

Comment:
Lines 586-595 we have:
```
    store_covariances : boolean
        If True the covariance matrices are computed and stored in the
        `self.covariances_` attribute.

        .. versionadded:: 0.17

    tol : float, optional, default 1.0e-4
        Threshold used for rank estimation.

        .. versionadded:: 0.17
```
These lines are incorrectly located in Attributes instead of Parameters.
Could you move them here?